diagnostic.md

```diff
@@ -36,19 +36,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+It is a relative path because it exists in our local file system instead of being hosted online somewhere.
```

Comment:
This is an absolute path, we can tell because `/Users/blah_blah/Desktop` uses the full path. It gives the entire roadmap to the location.